dbt_platform_helper/providers/load_balancers.py

```diff
@@ -34,18 +34,18 @@ def find_https_listener(session: boto3.Session, app: str, env: str) -> str:
         pass
 
     if not listener_arn:
-        raise ListenerNotFoundError()
+        raise ListenerNotFoundException()
 
     return listener_arn
 
 
-class LoadBalancerNotFoundError(Exception):
+class LoadBalancerNotFoundException(Exception):
```

Comment:
Don't these want to inherit from PlatformException or AWSException?

Reply:
Good shout. I created a `LoadBalancerException`. It may change, but that's a good enough grouping for pre refactor this bit of code.